README.md

```diff
@@ -5,4 +5,30 @@
 
 # Google Container Tools for IntelliJ
 
-In development
+This plugin adds support for [Kubernetes](https://www.kubernetes.io) development into [Jetbrains family of IDEs](https://www.jetbrains.com/products.html), including IntelliJ (both Community and Ultimate editions), GoLand, PyCharm, WebStorm (and others).
```

Comment:
into the* JetBrains family of IDEs

(also note the casing of JetBrains - should be updated in other locations too if its not camel cased).

Reply:
Good catch, thanks, updated in two places, other ones are URLs.